sktime/forecasting/compose/_reduce.py

```diff
@@ -1885,6 +1885,13 @@ class DirectReductionForecaster(BaseForecaster, _ReducerMixin):
         "panel" = second lowest level, one reduced model per panel level (-2)
         if there are 2 or less levels, "global" and "panel" result in the same
         if there is only 1 level (single time series), all three settings agree
+    windows_identical : bool, optional, default=True
+        Specifies whether all direct models use the same number of observations
+        (True: Total observations - maximum forecasting horizon due to
+        imputation of ReductionTransformer, otherwise: Total observations + 1 -
+        window_length + maximum forecasting horizon) or a different number of
+        observations (False: Total observations + 1 - window_length +
+        forecasting horizon).
```

Comment:
the docstring is not so clear. In the parenthesis where you explain what `True` does, you have also an `otherwise`, but then you explain what `False` does and it is different from what is done in the `otherwise` above

Reply:
Yes, I am aware. I used the docstring from `make_reduction` and amended it. The problem is that - from my perspective - `otherwise` is the expected behavior, but not currently implemented (see my comment about imputation in `ReductionTransformer` above). Happy to delete that part though.

Reply:
I've changed the docstring and formatted it to make it more readable.